scripts/db_migrator.py

```diff
@@ -1004,9 +1023,31 @@ def version_4_0_2(self):
     def version_4_0_3(self):
         """
         Version 4_0_3.
-        This is the latest version for master branch
         """
         log.log_info('Handling version_4_0_3')
+
+        sflow_tbl = self.configDB.get_table('SFLOW')
+        for k, v in sflow_tbl.items():
+            if 'sample_direction' not in v:
+                v['sample_direction'] = 'rx'
+                self.configDB.set_entry('SFLOW', k, v)
+
+        sflow_sess_tbl = self.configDB.get_table('SFLOW_SESSION')
+        for k, v in sflow_sess_tbl.items():
+            if 'sample_direction' not in v:
+                v['sample_direction'] = 'rx'
+                self.configDB.set_entry('SFLOW_SESSION', k, v)
```

Comment:
why cannot this be handled in sflow orch? Where sample_direction can be default set to 'rx'. I dont see a reason why we need db_migrator for this case.

Reply:
Please remove the db_migrator changes from this PR. If you think its required, please create a separate PR for db_migrator as this PR is overloaded.

Reply:
> why cannot this be handled in sflow orch? Where sample_direction can be default set to 'rx'. I dont see a reason why we need db_migrator for this case.

If not handled, when you upgrade with existing sflow configuration, "sample_direction" will not be seen in schema and for any new sflow configuration, "sample_direction" will be set. So there will be mismatch in sflow DB schema between old and new sflow entries which will create confusion to user.

Reply:
As suggested, removed db_migrator changes from this PR. Separate PR to be raised for db_migrator once this is PR merged.